src/security/cli_security.c

```diff
@@ -79,28 +87,38 @@ request_credentials_via_drpc(Drpc__Response **response)
 	if (request == NULL) {
 		D_ERROR("Couldn't allocate dRPC call\n");
 		drpc_close(agent_socket);
+		free(sock_path);
 		return -DER_NOMEM;
 	}
 
 	rc = drpc_call(agent_socket, R_SYNC, request, response);
 
 	drpc_close(agent_socket);
 	drpc_call_free(request);
+	free(sock_path);
 	return rc;
 }
 
 static char *
 get_agent_socket_path(void)
 {
+	char *path = NULL;
```

Comment:
Just a style thought - Probably don't want a comment breaking up variable declarations at the top.